docs(llm): proofread (#2905)
scaleway/docs-content: ai-data/llm-inference/quickstart.mdx

  - ia
---

> Scaleway LLM Inference service, the 1st European LLM Inference platform on the market, is a scalable and secure inference engine for large language models.

Reviewer: ```suggestion
Scaleway LLM Inference service, the first European LLM Inference platform on the market, is a scalable and secure inference engine for large language models.
```

Scaleway LLM Inference is a fully managed service that allows you to serve generative AI models in a production environment.
With Scaleway LLM Inference, you can easily deploy, manage, and scale LLMs without worrying about the underlying infrastructure.

Here are some of the key features of Scaleway Inference service:

* **Easy deployment**: Deploy state-of-the-art open weights LLMs with just a few clicks. Scaleway Inference provides a simple and intuitive interface for generating dedicated endpoints.
* **Security**: Scaleway provides a secure environment for running your models. Our platform is built on top of a secure architecture, and we use state-of-the-art security.

> * **Complete data privacy**: No storage nor third-party access to your data (prompt & responses), ensuring it remains exclusively yours and within Europe.

Reviewer: ```suggestion
* **Complete data privacy**: No storage or third-party access to your data (prompt or responses), to ensure it remains exclusively yours and stays within Europe.
```

> * **Auto-scaling (soon!)**: Scaleway LLM Inference automatically scales your instances based on the demand, ensuring that your models are always available and responsive.

Reviewer: ```suggestion
* **Auto-scaling (coming soon)**: Scaleway LLM Inference automatically scales your instances based on demand, ensuring that your models are always available and responsive.
```

Reviewer 2: Instances here should be capitalized? @RoRoJ

<Message type="important">

>   This service is in private beta. [Specific Terms and Conditions](https://www.scaleway.com/en/contracts/) are applicable.

Reviewer: ```suggestion
  This service is in private beta. [Terms and conditions](https://www.scaleway.com/en/contracts/) apply.
```

Author: It was meant to say we have specific terms as opposed to the general terms of services. In fact, the beta specific terms take precedence over our general terms of service.

Reviewer: OK I've added back the word specific

> 2. Click on **Create deployment** to launch the deployment creation wizard.

Reviewer 3: ```suggestion
2. Click **Create deployment** to launch the deployment creation wizard.
```

3. Provide the necessary information:
    - Select the desired model and the quantization to use for your deployment from the available options:
        - [Llama-2-70b](/ai-data/llm-inference/reference-content/llama-2-70b)
        - [Mixtral-8x7B-Instruct-v0.1](/ai-data/llm-inference/reference-content/mixtral-8x7b)
        - [WizardLM-70b-V1.0](/ai-data/llm-inference/reference-content/wizardlm-70b)
        <Message type="note">
          Some models may require acceptance of an end-user license agreement. If prompted, review the terms and conditions and accept the license accordingly.
        </Message>
    - Choose the geographical **region** for the deployment.
    - Specify the GPU Instance type to be used with your deployment.
4. Enter a **name** for the deployment, along with optional tags to aid in organization.
5. Configure the **network** settings for the deployment:
    - Enable **Private Network** for secure communication and restricted availability within Private Networks. Choose an existing Private Network from the drop-down list, if applicable.

>     - Enable **Public Network** to access resources via the public Internet. A token protection is enabled by default.

Reviewer: ```suggestion
    - Enable **Public Network** to access resources via the public Internet. Token protection is enabled by default.
```

5. Type **DELETE** to confirm and click **Delete deployment** to delete your deployment. 
 
<Message type="important">

>   Deleting a deployment is a permanent action, erasing all its associated configuration and resources.

Reviewer: ```suggestion
  Deleting a deployment is a permanent action, and erases all its associated configuration and resources.
```
